src/tests/ftest/harness/test_harness_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from random import choice
+
+from apricot import TestWithServers
+from general_utils import run_task, pcmd
+
+
+class TestHarnessTests(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Collection of tests to verify the behavior of the test harness.
+
+    :avocado: recursive
+    """
+
+    def test_core_files(self):
+        """Test to verify core file creation.
+
+        This test will send a signal 6 to a random daos_io_server process so
+        that it will create a core file, allowing the core file collection code
+        in launch.py to be tested.
+
+        :avocado: tags=medium,test_harness,core_files
```

Comment:
Doesn't `medium` also have to have `ib2`?  But more importantly, does this really need to run on a hardware cluster?

Reply:
I only added 'medium' as a reminder the test needs a minimum of 4 servers to run this test.  It does not need `hw` so I excluded the `hw,ib2` tags.  It was not my intention to run this test regularly - but we could, thoughts? - so I intentionally did not include any other tags.

Reply:
Ultimately, just thinking this could/should run on VMs, entirely due to our HW constraints.

Reply:
I can reduce the required host count.

Reply:
Any thoughts on whether this should be run regularly? daily? weekly?

Reply:
It's pretty cheap isn't it?  Maybe on PRs.  It would be interesting to run it first and bail out of the run if it fails.  Although it is just stacktrace gathering so maybe that is not justified.